lafleur/orchestrator.py

```diff
@@ -78,13 +79,14 @@ class LafleurOrchestrator:
     """
 
     def __init__(
-        self, fusil_path: str, min_corpus_files: int = 1, differential_testing: bool = False
+        self, fusil_path: str, min_corpus_files: int = 1, differential_testing: bool = False, timeout: int = 10
     ):
         """Initialize the orchestrator and the corpus manager."""
         self.differential_testing = differential_testing
         self.fusil_path = fusil_path
         self.ast_mutator = ASTMutator()
         self.boilerplate_code = None
+        self.timeout = timeout  # Store the timeout value
```

Comment:
This is correct. A few lines below, please change:
```python
        self.corpus_manager = CorpusManager(
            self.coverage_state, self.run_stats, fusil_path, self.get_boilerplate
        )
```
To:
```
        self.corpus_manager = CorpusManager(
            self.coverage_state, self.run_stats, fusil_path, self.get_boilerplate, self.timeout 
        )
```
So the `CorpusManager` gets the correct timeout value.